src/cljc/athens/util.cljc

```diff
@@ -10,3 +10,14 @@
   []
   (-> (js/Date.) .getTime))
 
+
+(defn scrollIfNeeded
+  [element container]
+  (if (< (.-offsetTop element) (.-scrollTop container))
+    (set! (.-scrollTop container) (.-offsetTop element))
```

Comment:
I like to use `(.. container -scrollTop)` for JS interop, even when I don't necessarily need to chain JS methods/properties. Doesn't matter too much whether we use `..` syntax sugar or not, but I'd like to be somewhat consistent